micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/SidebarAddEditItems.js

```
@@ -0,0 +1,537 @@
+            });
+
+    }, [])
+
+    const { data: USER_ROLE } = Digit.Hooks.useCustomMDMS(tenantId, "ACCESSCONTROL-ROLES", [{ name: "roles" }], {
+        config: {
+            cacheTime: 0,
+            staleTime: 0,
+        },
+        select: (data) => {
+            return data;
+            ;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)